rke/templates/templates.go

```diff
@@ -46,6 +46,7 @@ const (
 	coreDnsv18  = "coredns-v1.8"
 	coreDnsv116 = "coredns-v1.16"
 	coreDnsv117 = "coredns-v1.17"
+	coreDnsv183 = "corends-v1.8.3"
```

Comment:
typo `corends`, als the numbers are corresponding to k8s versions but I guess thats not the same everywhere

Reply:
oops thanks! yeah it's not the same, this template would also work with 1.20.x + rbac rules changed for addon version so thought of going with 1.8.3